flink-runtime/src/main/java/org/apache/flink/runtime/minicluster/MiniCluster.java

```diff
@@ -1175,6 +1193,7 @@ protected RpcService createLocalRpcService(Configuration configuration, RpcSyste
     private void terminateMiniClusterServices() throws Exception {
         // collect the first exception, but continue and add all successive
         // exceptions as suppressed
+        LOG.info("term start");
```

Comment:
What is the purpose of logging here? It feels like debug logging but could you clarify if not?